sys/kern/imgact_elf.c

```diff
@@ -1642,11 +1650,13 @@ static void note_procstat_vmmap(void *, struct sbuf *, size_t *);
  * Write out a core segment to the compression stream.
  */
 static int
-compress_chunk(struct coredump_params *p, char *base, char *buf, u_int len)
+compress_chunk(struct coredump_params *p, char *basev, char *buf, u_int len)
```

Comment:
Why `basev`?

Reply:
"base virtual address".  Renaming the parameter makes the diff in the rest of the function smaller.

Reply:
`baseva` or `base_vaddr` might be more readable?

Reply:
Changed to `base_vaddr`.